### PR TITLE
Add excludes to Baffle options

### LINKED ISSUE #16: Expose the exclude option for Obfuscator.render

### PR DESCRIPTION
**Pull request by danreeves**
Fixes #16

### CODE REVIEW

**Comment by camwiegert** [comment]
I think exposing this makes sense. Thanks! The plumbing is already there, so we might as well. I'll review and merge this soon.

File: src/baffle.js [hunk 1]
```diff
@@ -8,7 +8,8 @@ import Obfuscator from './obfuscator';
 
 const defaults = {
     characters: 'AaBbCcDdEeFfGgHhIiJjKkLlMmNnOoPpQqRrSsTtUuVvWwXxYyZz~!@#$%^&*()-+=[]{}|;:,./<>?'.split(''),
-    speed: 50
+    speed: 50,
+    exclude: [' '],
```
Comment (camwiegert) [inline src/baffle.js]: Could we drop the trailing comma? And maybe reorder to keep speed last (alphabetical) while we're at it?

**Comment by danreeves** [comment]
Certainly!

**Comment by danreeves** [comment]
The changes have been made btw

**Comment by camwiegert** [comment]
Just a heads up that I plan to merge this very soon. I'm waiting until I have a moment to update documentation on [the site](https://camwiegert.github.io/baffle/#options).

**Comment by danreeves** [comment]
#22

**Pull request merged**